Rakefile

```diff
@@ -2654,6 +2654,7 @@ def copy_sample_files
                   'base-dhw-desuperheater-gshp.xml',
                   'base-dhw-desuperheater-tankless.xml',
                   'base-dhw-indirect-outside.xml',
+                  'base-dhw-indirect-standbyloss.xml',
```

Comment:
I excluded standbyloss test file in ERI while tested locally its rated home configuration. Is it good enough? 
Another option is adding this test file and checking rated home configurations with a new test in 301EnergyRatingIndexRuleset/tests/test_water_heating.rb

Reply:
It would be great if you could do the latter. You should be able to copy/paste the existing "test_indirect_water_heating" test.